Data Prepper documentation updates: autogeneration campaign (#7707)
opensearch-project/documentation-website: _data-prepper/pipelines/configuration/processors/convert_entry_type.md

You can configure the `convert_entry_type` processor with the following options.

<!--
This table is autogenerated. Do not edit it.
- name: convert_entry_type
- pluginType: processor
- source: https://github.com/opensearch-project/data-prepper/blob/7d15115c281687aab50e5c471fd210cb1ef90fc5/data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/ConvertEntryTypeProcessorConfig.java
-->

| Option | Required | Description |
| :--- | :--- | :--- |
| `key`| Yes | Key whose value needs to be converted to a different type. |
| `keys`| Yes | Keys whose value needs to be converted to a different type. |
| `type` | No | Target type for the key-value pair. Possible values are `integer`, `long`, `double`, `big_decimal`, `string`, and `boolean`. Default value is `integer`. |

> | `null_values` | No | String representation of what constitutes a `null` value. If the field value equals one of these strings, then value is considered `null` and is converted to `null`. |

Reviewer: ```suggestion
| `null_values` | No | String representation of what constitutes a `null` value. If the field value equals one of these strings, then the value is considered `null` and is converted to `null`. |
```

> | `scale` | No | Modifies the scale when used to convert to a `big_decimal`. The default value is `0`. |

Reviewer: Is this saying that scale is only modified when the option is converted to `big_decimal`?

Author: I just pushed a new commit that attempts to correct the wording.

We are modifying the scale of the `big_decimal` value itself. This field is only applicable for `big_decimal`.

> | `tags_on_failure` | No | A list of tags to add to the event metadata when the event fails to convert. |

Reviewer: ```suggestion
| `tags_on_failure` | No | A list of tags to be added to the event metadata when the event fails to convert. |
```

> | `convert_when` | No | Specifies a condition for performing the `convert_entry_type` operation using a [Data Prepper expression]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/expression-syntax/). If specified, the `convert_entry_type` operation will only run when the expression evaluates to true. |

Reviewer: ```suggestion
| `convert_when` | No | Specifies a condition for performing the `convert_entry_type` operation using a [Data Prepper expression]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/expression-syntax/). If specified, the `convert_entry_type` operation will only run when the expression evaluates to `true`. |
```
